MergeDatiJinja.py

```diff
@@ -1,5 +1,5 @@
 from jinja2 import Template
-with open('ConfTemplate.txt','r') as fp:
```

Comment:
why did you rename this file? As you have done this script doesn't work. Should you maybe change the line about read file?

Reply:
I changed it just to make it clear that it is a Jinja2 template.
It is working for me, the extension of the file shouldn't matter when parsing